DotNetRu.Clients.UI/App.xaml.cs

```diff
@@ -37,21 +38,22 @@ public App()
             ViewModelBase.CurrentLanguage = AppResources.Culture.Name.Substring(0, 2);
             this.InitializeComponent();
             ViewModelBase.Init();
+            this.MainPage = new BottomTabbedPage();
             // The root page of your application
-            switch (Device.RuntimePlatform)
-            {
-                case Device.Android:
-                    this.MainPage = new RootPageAndroid();
-                    break;
-                case Device.iOS:
-                    this.MainPage = new EvolveNavigationPage(new RootPageiOS());
-                    break;
-                case Device.UWP:
-                    this.MainPage = new RootPageWindows();
-                    break;
-                default:
-                    throw new NotImplementedException();
-            }
+            //switch (Device.RuntimePlatform)
```

Comment:
Please remove - it's no longer needed. BTW, is BottomTabbedPage supported on Windows 10?

Reply:
it will render in default tabbed page on windows